### PR TITLE
Pull request for tutorial pages update

### PR DESCRIPTION
**Pull request by giuluck**

### CODE REVIEW

File: pages/tutorial/biochemistry.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+{% endhighlight %}
+
+Then, when creating the cells, you must use these specific implementations:
+{% highlight yaml %}
+nodes:
+  type: CircularDeformableCellImpl
+  parameters: [max-radius, rigidity]
+{% endhighlight %}
+
+The min radius of the cell is so that ``min-radius = rigidity * max-radius`` and the two parameters are used to compute collisions and impacts between the cells.
```
Comment (DanySK) [inline pages/tutorial/biochemistry.md]: min -> minimum

File: pages/tutorial/cli.md [hunk 2]
```diff
@@ -12,14 +12,13 @@ header:
    image_fullwidth: "header_mol.png"
 ---
 
-This section will explain the possible ways to launch simulations
-
-### Using the redistributable jar and the command line
-
 ### Using Gradle
 
+You can find tutorials about how to setup a project with gradle and how to create configuration files for all the incarnations at the [Github's Alchemist Page][github]
```
Comment (DanySK) [inline pages/tutorial/cli.md]: all the incarnations at the [Github's Alchemist Page][github] -> any incarnation [here][github]

File: pages/tutorial/gui.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+header:
+   image_fullwidth: "header_mol.png"
+---
+
+## Starting Alchemist in graphical mode
+
+### Prerequisites
+
+* A working graphical environment
+* A Java 8 (or newer) runtime environment
```
Comment (DanySK) [inline pages/tutorial/gui.md]: Java 8 with OpenJFX 8, Java 11, or a later Java version. Java 9 and Java 10 are not supported, although the Java 8 version of Alchemist might work, if OpenJFX or another JavaFX library is correctly installed..

File: pages/tutorial/gui.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+### Download and installation
+
+Alchemist does not require any installation procedure. It is delivered in form of a runnable jar file, that can be retrieved from [the Alchemist release page on Github](https://github.com/AlchemistSimulator/alchemist/releases). The file name should match ``alchemist-redist-VERSION.jar`` where ``VERSION`` is a [SemVer](http://semver.org/spec/v2.0.0.html) version.
+
+#### Launch from the graphical interface
+
+Once downloaded, if your Java 8 environment is correctly setup, you should be able to launch it the same way you open any file (classically in most environments, by double clicking it). If Alchemist does not start this way, you can fall back to using a terminal.
```
Comment (DanySK) [inline pages/tutorial/gui.md]: Java 8 -> Java

File: pages/tutorial/gui.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+## Starting Alchemist in graphical mode
+
+### Prerequisites
+
+* A working graphical environment
+* A Java 8 (or newer) runtime environment
+
+### Download and installation
+
+Alchemist does not require any installation procedure. It is delivered in form of a runnable jar file, that can be retrieved from [the Alchemist release page on Github](https://github.com/AlchemistSimulator/alchemist/releases). The file name should match ``alchemist-redist-VERSION.jar`` where ``VERSION`` is a [SemVer](http://semver.org/spec/v2.0.0.html) version.
```
Comment (DanySK) [inline pages/tutorial/gui.md]: Please make this consistent with the fact that we offer a way to configure the simulator via Gradle

File: pages/tutorial/index.md [hunk 6]
```diff
@@ -17,6 +17,7 @@ header:
 0. [Simulator setup][setup]
 0. [How to write simulations][Simulations]
 0. [The graphical interface][gui]
+0. [YAML for Alchemist][yaml]
```
Comment (DanySK) [inline pages/tutorial/index.md:20]: Why is this different than "How to write simulations"?

**Comment by DanySK** [comment]
@giuluck I'm waiting for updates

File: pages/tutorial/index.md (hunk 6 above)
Comment (DanySK) [reply pages/tutorial/index.md:20]: Does this still need to get fixed?

Comment (giuluck) [reply pages/tutorial/index.md:20]: I removed the "YAML for Alchemist" entry, now there is no reference to it but the page is still in the repository.
However, I did not change the "How to write simulation" page.
If needed, I can add some examples taken from the tutorials at the end of every section or even link the tutorial repositories.
Moreover, I can join the two pages and, then, remove the not used one.

**Comment by DanySK** [comment]
Looks good to go

**Pull request merged**